README.md

```diff
@@ -7,4 +7,17 @@ A minimal web browser made with Material Design, Electron and React.
 Have a look at the `CONTRIBUTING.md` file in order to adhere to the codebase and
 commit format.
 
+## Environments variables
```

Comment:
I'd add this section in the `CONTRIBUTING.md` file, since it's strictly related to the contribution phase.